tests/data_association/test_data_association.py

```diff
@@ -138,7 +138,12 @@ def verify_triangulation_sharedCal_2poses(self, triangulation_mode: Triangulatio
         )
         da = DataAssociation(min_track_len=3, triangulation_options=triangulation_options)
         triangulated_landmark_map, _ = da.run(
-            len(cameras), cameras, matches_dict, keypoints_list, cameras_gt=[None] * len(cameras)
+            len(cameras),
+            cameras,
+            matches_dict,
+            keypoints_list,
+            cameras_gt=[None] * len(cameras),
+            relative_pose_priors={},
```

Comment:
Oh, we can just do that? I thought dask needed dict of same size. In that case, please feel free to simplify the `test_shonan` code